### PR TITLE
DOCS-5104 Exchange: full text search

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 1]
```diff
@@ -9,6 +9,7 @@ endif::[]
 
 * When using direct upload, provide relevant information about the files that should be uploaded for each asset type. (EXC-3427)
 * Users can now customize the order of the user-generated pages in the public portal. (EXC-3405)
+* Exchange now searches the full text of API specifications and portal content. (EXC-2967)
```
Comment (nicolasgnr) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Are we indexing the full API Spec text? @wwarner-salesforce @LucasLudueno @sancheztavo

Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Is it only certain fields in the API spec? If so, should I change this to "Exchange now searches API specifications and portal content."?

Comment (mcampo) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]:
```suggestion
* Exchange search now looks for terms in asset portal pages content (full-text search) and in properties defined in the API specification (for RAML and OAS APIs). Learn more (link to updated docs) (EXC-2967)
```

Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Fixed.

Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Updated. Thank you!

**Comment by wwarner-salesforce** [comment]
Merge planned for Monday per Mariano. Goes with https://github.com/mulesoft/docs-release-notes/pull/760 .

**Pull request merged**